### PR TITLE
Binding LIGHT_PARAM_INTENSITY in RenderingServer

### LINKED ISSUE #78570: RenderingServer.LIGHT_PARAM_INTENSITY is not accessible to GDScript
### Godot version

4.0.3-stable

### System information

Linux 6.3.6-arch1-1

### Issue description

When using RenderingServer.light_set_param, the enum LIGHT_PARAM_INTESNITY is missing. It's usable in c++, as seen in [godot/scene/3d/light_3d.h](https://github.com/godotengine/godot/blob/e74bf831c2d3ece79b849405f03999281f807648/scene/3d/light_3d.h#L61), but isn't bound for GDScript in [godot/servers/rendering_server.cpp](https://github.com/godotengine/godot/blob/e74bf831c2d3ece79b849405f03999281f807648/servers/rendering_server.cpp#L1916) with the other LightParam enum values.

### Steps to reproduce

This code produces the error: Cannot find member "LIGHT_PARAM_INTENSITY" in base "RenderingServer".
```

var light_RID = RenderingServer.spot_light_create()
RenderingServer.light_set_param(light_RID, RenderingServer.LIGHT_PARAM_INTENSITY, 10)

```

### Minimal reproduction project

[MissingEnum.zip](https://github.com/godotengine/godot/files/11835811/MissingEnum.zip)

### PR DESCRIPTION
**Pull request by brno32**
Fixes https://github.com/godotengine/godot/issues/78570

### CODE REVIEW

File: doc/classes/RenderingServer.xml [hunk 1]
```diff
@@ -4272,6 +4272,8 @@
 		</constant>
 		<constant name="LIGHT_PARAM_TRANSMITTANCE_BIAS" value="19" enum="LightParam">
 		</constant>
+		<constant name="LIGHT_PARAM_INTENSITY" value="20" enum="LightParam">
+		</constant>
```
Comment (AThousandShips) [inline doc/classes/RenderingServer.xml:4276]: Please describe the property if you are able, probably good hint from the relevant property in light

**Review by AThousandShips — APPROVED** [review]
LGTM, description seems clear, not familiar with the actual setting so TIWAGOS

**Comment by brno32** [comment]
> LGTM, description seems clear, not familiar with the actual setting so TIWAGOS

Thanks!

Would you mind explaining to me what TIWAGOS means? I can't find it online 🤣

**Comment by akien-mga** [comment]
> It's a Godot specific one it seems, Take It With A Grain Of Salt, think YuriSizov originated that one

RandomShaper actually ;) https://github.com/godotengine/godot/pull/77608#pullrequestreview-1450700418

File: doc/classes/RenderingServer.xml [hunk 2]
```diff
@@ -4272,6 +4272,9 @@
 		</constant>
 		<constant name="LIGHT_PARAM_TRANSMITTANCE_BIAS" value="19" enum="LightParam">
 		</constant>
+		<constant name="LIGHT_PARAM_INTENSITY" value="20" enum="LightParam">
+			The intensity of the light source measured in Lumens. Typically used when [member ProjectSettings.rendering/lights_and_shadows/use_physical_light_units] is [code]true[/code] to determine the total energy of a given light.
```
Comment (clayjohn) [inline doc/classes/RenderingServer.xml:4276]: This isn't quite accurate, intensity can be measured in lumens (when the light is a spot light or an omni light) or lux (when the light is a directional light). Additionally, the setting only has an effect when ``ProjectSettings.rendering/lights_and_shadows/use_physical_light_units`` is true

For reference, the description in Light3D is:

> Constant for accessing [member light_intensity_lumens] and [member light_intensity_lux]. Only used when [member ProjectSettings.rendering/lights_and_shadows/use_physical_light_units] is [code]true[/code].

**Comment by YuriSizov** [comment]
> > It's a Godot specific one it seems, Take It With A Grain Of Salt, think YuriSizov originated that one
> 
> RandomShaper actually ;) [#77608 (review)](https://github.com/godotengine/godot/pull/77608#pullrequestreview-1450700418)

I popularized it though!

File: doc/classes/RenderingServer.xml (hunk 2 above)
Comment (brno32) [reply doc/classes/RenderingServer.xml]: thank you for this insight!

**Review by akien-mga — APPROVED** [review]

**Comment by akien-mga** [comment]
Thanks!

**Pull request merged**